Marlin/Configuration_adv.h

```diff
@@ -2206,6 +2201,15 @@
   //#define TFT_BTOKMENU_COLOR 0x145F // 00010 100010 11111 Cyan
 #endif
 
+//
+// LCD Backlight Timeout
+// Requires a display with a controllable backlight
+//
+//#define LCD_BACKLIGHT_TIMEOUT_MINS 1  // (minutes) Timeout before turning off the backlight
+#if defined(DISPLAY_SLEEP_MINUTES) || defined(LCD_BACKLIGHT_TIMEOUT_MINS)
+  #define EDITABLE_DISPLAY_TIMEOUT      // Edit timeout with M255 S<minutes> and a menu item
+#endif
+
 //
 // ADC Button Debounce
 //
```

Comment:
@thinkyhead @ line 2236 you see `// @section lcd` followed by `BABYSTEPPING`, yet this whole section above *IS* `@section lcd`. so should `// @section lcd` below this be there?